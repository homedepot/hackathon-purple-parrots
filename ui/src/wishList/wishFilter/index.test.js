import WishFilter from './index'
import React from 'react'
import { shallow } from 'enzyme'

describe('WishFilter tests', () => {
  it('snapshot', () => {
    const wrapper = shallow(<WishFilter/>)

    expect(wrapper).toMatchSnapshot()
  })
})
