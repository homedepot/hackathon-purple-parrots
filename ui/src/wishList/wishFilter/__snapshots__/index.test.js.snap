// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WishFilter tests snapshot 1`] = `ShallowWrapper {}`;
